docs/cudf/source/user_guide/10min.ipynb

```diff
@@ -2,20 +2,25 @@
  "cells": [
```

Comment:
<p>Same as above about the absolute paths in nvidia-smi output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='170'/>